src/components/service-status-badge/docs.mdx

```diff
@@ -0,0 +1,30 @@
+---
+componentName: ServiceStatusBadge
+---
+
+## Overview
```

Comment:
Added a `docs.mdx` for `ServiceStatusBadge` as a way to show all the variations, since the live status is usually only useful to preview the `operational` variation.

Reply:
This is great